### PR TITLE
Add Pipelines as Code for build-service

### PR DESCRIPTION
**Pull request by mbharatk**

### CODE REVIEW

File: components/build/build-service/.tekton/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+resources:
+- serviceaccount.yaml
+- pipelines-as-code.yaml
+- pvc.yaml
+
+namespace: build-service-system
```
Comment (mmorhun) [inline components/build/build-service/.tekton/kustomization.yaml:6]: I think the namespace should be `build-service`.
The default `build-service-system` is generated by operator-sdk, but in our components we don't usually use the `system` suffix: `application-service` namespace is an example.

File: components/build/build-service/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: build-service-system
```
Comment (mmorhun) [inline components/build/build-service/namespace.yaml:4]: Same as above

File: components/build/build-service/.tekton/pipelines-as-code.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
+metadata:
+  name: build-service-pac
+  namespace: build-service-system
```
Comment (mmorhun) [inline components/build/build-service/.tekton/pipelines-as-code.yaml:5]: We missed it last time. Should be consistent.

Comment (mbharatk) [reply components/build/build-service/.tekton/pipelines-as-code.yaml]: Done

**Review by Michkov — APPROVED** [review]

**Pull request merged**